crates/subspace-farmer/src/commitments.rs

```diff
@@ -133,7 +133,7 @@ impl Commitments {
                     .collect();
 
                 for (tag, index) in tags.iter().zip(indexes) {
-                    db.put(tag, index.to_le_bytes())
```

Comment:
Why did you change `to_le_bytes()` to `to_be_bytes()` in this file?

Reply:
For consistency as, there are places where was `le` and some place where were `be`.

Was there a reason why it was little endian?

Reply:
Yes, most modern processors (including x86 and most ARM-based) are little-endian, hence `to/from_le_bytes()` is essentially free (it is the way it is implemented in memory already), hence should generally be the default choice.

`to_be_bytes` is used for tags specifically because that allows us to use range searches in RocksDB database directly, we treat tags as 64-bit numbers, which they are, but only in big-endian encoding they are sorted on byte level in ascending order.

Generally, please don't refactor such things when you're working on unrelated things without full context, it just makes PR bigger and review longer. Refactoring separately, new features separately is a preferred approach.

Reply:
Thanks for clarification!
Should we put those kind of things in types? It would be very handy to also document it in all the places there specific endianness is used for a specific reason.

Reply:
I think so, but not that type aliases are not real types. Eventually I'd like to do that, maybe as a separate PR. We need to asses carefully what makes sense to put into new types and where (something like `PieceIndex` is generic and should likely be in `subspace-core-primitives`, while offset is farmer-specific implementation detail).